### ISSUE TITLE
[Website] Ensures every HTML document has a lang attribute.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6427]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/connect/ 9/14/2021

### Application Name

Website

### Problem Description

[ADO 36106889](https://microsoft.visualstudio.com/OS/_workitems/edit/36106889)

Repro Steps:
1. Launch the above URL.
2. Now, run the accessibility insights for web.
3. Observe that HTML element does not have a lang attribute.

Actual Result:
HTML element does not have a lang attribute under connect web page.

Expected Result:
Every element of HTML document should have lang attribute.

### Card JSON

```JSON
No Card
```

**Comment by paulcam206** [comment]
This page should be removed altogether as it is no longer necessary.

**Comment by PankajBhojwani** [comment]
Closing this as the `adaptivecards.io/connect` page has now been removed
